src/layout/TabbedContent.js

```diff
@@ -0,0 +1,135 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "src/common/HTMLWidget", "src/api/IInput", "src/layout/Surface", "src/chart/Pie", "src/chart/MultiChart", "src/google/Line", "css!./TabbedContent"], factory);
+    } else {
+        root.form_Form = factory(root.d3, root.common_HTMLWidget, root.layout_IInput, root.layout_Surface, root.chart_Pie, root.chart_MultiChart, root.chart_Line);
+    }
+}(this, function (d3, HTMLWidget, IInput, Surface, Pie, MultiChart, Line) {
+    function TabbedContent() {
+        HTMLWidget.call(this);
+
+        this._tag = "div";
+        this._tabButtons = [];
+    }
+    TabbedContent.prototype = Object.create(HTMLWidget.prototype);
+    TabbedContent.prototype._class += " layout_TabbedContent";
+
+    TabbedContent.prototype.publish("width", null, "number", "Popup content width",null,{});
+    TabbedContent.prototype.publish("height", null, "number", "Popup content height",null,{});
```

Comment:
No need to publish width and height (and they are used in the base class) - this widget should always expand to fit the div it is being placed into (like all the other widgets).